pkg/minikube/constants/constants.go

```diff
@@ -44,7 +43,7 @@ const (
 	DefaultDiskSize = "20g"
 )
 
-var DefaultIsoUrl = "https://github.com/minishift/minishift/releases/download/v" + version.GetVersion() + "/boot2docker.iso"
+var DefaultIsoUrl = "https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.0/minishift-b2d.iso"
```

Comment:
Does hardcoded verison make sense can't we add a go function to get latest release url? I have prototype in python which I can put in go also : https://gist.github.com/praveenkumar/f5e26a20f2b6a6640b86bc2934371bf9

@hferentschik wdyt?